spec/post_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  it 'is not valid without a title' do
+    post = Post.new(title: nil)
+    expect(post).to_not be_valid
+  end
+
+  it 'is not valid with a title longer than 250 characters' do
+    post = Post.new(title: 'a' * 251)
+    expect(post).to_not be_valid
+  end
+
+  describe '#update_posts_counter' do
+    let!(:user) { User.create!(name: 'Test User', posts_counter: 0) }
+    let!(:post) { Post.create!(title: 'Test Post', author: user, comments_counter: 0, likes_counter: 0) }
+
+    it 'updates the posts counter of the author' do
+      expect { post.destroy }.to change { user.reload.posts_counter }.by(-1)
+    end
+  end
```

Comment:
- [OPTIONAL ] It may be better to also add the validation test to check if:
* CommentsCounter is an integer greater than or equal to zero. ✔️ 
* LikesCounter is an integer greater than or equal to zero. ✔️ 
😄